### ISSUE TITLE
Right click to open a process in a new tab or window

### ISSUE THREAD

**Issue by knudsvik** [equinor/flyt#279]
**User input**
While browsing through list of Flyts it would be great to "control-click" or right click to open flyts in a new tab to not loose position when navigating through multiple sheets.

**Sharepoint list ID**
96

**Comment by SjoenH** [comment]
Might be a quick fix. Need to investigate a bit https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

**Comment by nesadrian** [comment]
Moving to blocked because it is dependent on #567 to be merged to main

**Comment by nesadrian** [comment]
@QiJin-Bouvet This is ready for test in [https://test.flyt.equinor.com](https://test.flyt.equinor.com)

**Comment by QiJin-Bouvet** [comment]
Open process in new tab and new window works fine in TEST env.

**Comment by QiJin-Bouvet** [comment]
@nesadrian This feature is merged into main branch.
Will it be deployed to qa-graph-poc environment?

**Comment by nesadrian** [comment]
@QiJin-Bouvet Yes, I will do a merge soon
